src/dataSources/WikibaseActionApi.js

```diff
@@ -0,0 +1,20 @@
+const { RESTDataSource } = require('apollo-datasource-rest');
+
+module.exports = class WikibaseActionApi extends RESTDataSource {
+
+  constructor(baseUrl) {
+    super();
+    this.baseURL = baseUrl;
+  }
+
+  async getEntity(id) {
+    const result = await this.get('', {
+      action: 'wbgetentities',
+      format: 'json',
+      ids: [id]
```

Comment:
hmm, does that mean we can get multiple Items as well?

Reply:
Yup, wbgetentities can get data for multiple entities at once